build failed latest zig (git version)

error log
```
install
└─ install arocc
   └─ zig build-exe arocc ReleaseFast native 5 errors
src/aro/Compilation.zig:669:33: error: no field named 'aarch64_32' in enum 'Target.Cpu.Arch'
        .aarch64, .aarch64_be, .aarch64_32 => .{
                               ~^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:982:22: note: enum declared here
    pub const Arch = enum {
                     ^~~~
referenced by:
    generateBuiltinMacros: src/aro/Compilation.zig:530:34
    main__anon_9314: src/aro/Driver.zig:592:53
    3 reference(s) hidden; use '-freference-trace=5' to see all references
src/aro/target.zig:140:20: error: no field named 'aarch64_32' in enum 'Target.Cpu.Arch'
        .aarch64, .aarch64_32, .aarch64_be => 4,
                  ~^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:982:22: note: enum declared here
    pub const Arch = enum {
                     ^~~~
src/aro/target.zig:486:10: error: no field named 'aarch64_32' in enum 'Target.Cpu.Arch'
        .aarch64_32,
        ~^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:982:22: note: enum declared here
    pub const Arch = enum {
                     ^~~~
src/aro/target.zig:551:10: error: no field named 'aarch64_32' in enum 'Target.Cpu.Arch'
        .aarch64_32 => copy.cpu.arch = .aarch64,
        ~^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:982:22: note: enum declared here
    pub const Arch = enum {
                     ^~~~
src/aro/target.zig:586:10: error: no field named 'aarch64_32' in enum 'Target.Cpu.Arch'
        .aarch64_32 => "aarch64_32",
        ~^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:982:22: note: enum declared here
    pub const Arch = enum {
                     ^~~~
error: the following command failed with 5 compilation errors:
/usr/lib/zig/9999/bin/zig build-exe -fsingle-threaded -OReleaseFast --dep aro -Mroot=/home/misile/arocc/src/main.zig --dep system_defaults --dep build_options --dep backend --dep Builtins/Builtin.def --dep Attribute/names.def --dep Diagnostics/messages.def -Maro=/home/misile/arocc/src/aro.zig -Msystem_defaults=/home/misile/arocc/.zig-cache/c/3d9794cda06e6a27dedea6c15b23173e/options.zig -Mbuild_options=/home/misile/arocc/.zig-cache/c/b95a01851fd92449089fe4bbf9631212/options.zig --dep zig --dep build_options -Mbackend=/home/misile/arocc/src/backend.zig -MBuiltins/Builtin.def=/home/misile/arocc/.zig-cache/o/069591fa3ccedaa7b2294e77795fecd4/Builtin.zig -MAttribute/names.def=/home/misile/arocc/.zig-cache/o/082dfe4cf1b80947328d8fd5a9ce5b45/names.zig -MDiagnostics/messages.def=/home/misile/arocc/.zig-cache/o/bb71dae0c7373c62826b4179104e2297/messages.zig -Mzig=/home/misile/arocc/deps/zig/lib.zig -lc --cache-dir /home/misile/arocc/.zig-cache --global-cache-dir /home/misile/.cache/zig --name arocc --zig-lib-dir /usr/lib/zig/9999/lib/ --listen=-
Build Summary: 6/9 steps succeeded; 1 failed
install transitive failure
└─ install arocc transitive failure
   └─ zig build-exe arocc ReleaseFast native 5 errors
error: the following build command failed with exit code 1:
/home/misile/arocc/.zig-cache/o/cee061bb958e6d9b8a5f9d8462ed4a55/build /usr/lib/zig/9999/bin/zig /usr/lib/zig/9999/lib /home/misile/arocc /home/misile/arocc/.zig-cache /home/misile/.cache/zig --seed 0xd00cf1e5 -Z4d7a9dd63556e202 -Doptimize=ReleaseFast
```
related commit: https://github.com/ziglang/zig/commit/d1d95294fd657f771657ea671a6984b860347fb0
fastfetch
```
         -/oyddmdhs+:.                 misile@srt
     -odNMMMMMMMMNNmhy+-`              ----------
   -yNMMMMMMMMMMMNNNmmdhy+-            OS: Gentoo 2.15 x86_64
 `omMMMMMMMMMMMMNmdmmmmddhhy/`         Host: KVM/QEMU Standard PC (Q35 + ICH9, 2009) (pc-q35-9.0)
 omMMMMMMMMMMMNhhyyyohmdddhhhdo`       Kernel: Linux 6.10.2-gentoo-veritas
.ydMMMMMMMMMMdhs++so/smdddhhhhdm+`     Uptime: 15 hours, 23 mins
 oyhdmNMMMMMMMNdyooydmddddhhhhyhNd.    Packages: 394 (emerge)
  :oyhhdNNMMMMMMMNNNmmdddhhhhhyymMh    Shell: bash 5.2.26
    .:+sydNMMMMMNNNmmmdddhhhhhhmMmy    Display (QEMU Monitor): 1280x800 @ 75 Hz in 15″
       /mMMMMMMNNNmmmdddhhhhhmMNhs:    Terminal: sshd-session
    `oNMMMMMMMNNNmmmddddhhdmMNhs+`     CPU: Intel(R) Xeon(R) E5-2670 v3 (24) @ 2.29 GHz
  `sNMMMMMMMMNNNmmmdddddmNMmhs/.       GPU: RedHat Virtio 1.0 GPU
 /NMMMMMMMMNNNNmmmdddmNMNdso:`         Memory: 6.00 GiB / 47.02 GiB (13%)
+MMMMMMMNNNNNmmmmdmNMNdso/-            Swap: 0 B / 16.00 GiB (0%)
yMMNNNNNNNmmmmmNNMmhs+/-`              Disk (/): 20.00 GiB / 83.00 GiB (24%) - btrfs
/hMMNNNNNNNNMNdhs++/-`                 Local IP (enp6s18): 192.168.0.150/24
`/ohdmmddhys+++/:.`                    Locale: C.UTF-8
  `-//////:--.
```

I just try to delete `aarch64_32` with this
```patch
diff --git a/src/aro/target.zig b/src/aro/target.zig
index 81ba488..f844529 100644
--- a/src/aro/target.zig
+++ b/src/aro/target.zig
@@ -137,7 +137,7 @@ pub fn float80Type(target: std.Target) ?Type {
 pub fn defaultFunctionAlignment(target: std.Target) u8 {
     return switch (target.cpu.arch) {
         .arm, .armeb => 4,
-        .aarch64, .aarch64_32, .aarch64_be => 4,
+        .aarch64, .aarch64_be => 4,
         .sparc, .sparcel, .sparc64 => 4,
         .riscv64 => 2,
         else => 1,
@@ -327,7 +327,6 @@ pub const FPSemantics = enum {
     pub fn halfPrecisionType(target: std.Target) ?FPSemantics {
         switch (target.cpu.arch) {
             .aarch64,
-            .aarch64_32,
             .aarch64_be,
             .arm,
             .armeb,
@@ -483,7 +482,6 @@ pub fn get32BitArchVariant(target: std.Target) ?std.Target {
         .kalimba,
         .lanai,
         .wasm32,
-        .aarch64_32,
         .spirv,
         .spirv32,
         .loongarch32,
@@ -548,7 +546,6 @@ pub fn get64BitArchVariant(target: std.Target) ?std.Target {
         .x86_64,
         => {}, // Already 64 bit

-        .aarch64_32 => copy.cpu.arch = .aarch64,
         .arm => copy.cpu.arch = .aarch64,
         .armeb => copy.cpu.arch = .aarch64_be,
         .loongarch32 => copy.cpu.arch = .loongarch64,
@@ -583,7 +580,6 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
         .armeb => "armeb",
         .aarch64 => "aarch64",
         .aarch64_be => "aarch64_be",
-        .aarch64_32 => "aarch64_32",
         .arc => "arc",
         .avr => "avr",
         .bpfel => "bpfel",
```
and it has another error
```
install
└─ install arocc
   └─ zig build-exe arocc ReleaseFast native 2 errors
src/aro/Compilation.zig:669:33: error: no field named 'aarch64_32' in enum 'Target.Cpu.Arch'
        .aarch64, .aarch64_be, .aarch64_32 => .{
                               ~^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:982:22: note: enum declared here
    pub const Arch = enum {
                     ^~~~
referenced by:
    generateBuiltinMacros: src/aro/Compilation.zig:530:34
    main__anon_9314: src/aro/Driver.zig:592:53
    3 reference(s) hidden; use '-freference-trace=5' to see all references
src/aro/target.zig:694:22: error: switch must handle all possibilities
    const llvm_abi = switch (target.abi) {
                     ^~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:634:5: note: unhandled enumeration value: 'ilp32'
    ilp32,
    ^~~~~
/usr/lib/zig/9999/lib/std/Target.zig:620:17: note: enum 'Target.Abi' declared here
pub const Abi = enum {
                ^~~~
error: the following command failed with 2 compilation errors:
/usr/lib/zig/9999/bin/zig build-exe -fsingle-threaded -OReleaseFast --dep aro -Mroot=/home/misile/arocc/src/main.zig --dep system_defaults --dep build_options --dep backend --dep Builtins/Builtin.def --dep Attribute/names.def --dep Diagnostics/messages.def -Maro=/home/misile/arocc/src/aro.zig -Msystem_defaults=/home/misile/arocc/.zig-cache/c/3d9794cda06e6a27dedea6c15b23173e/options.zig -Mbuild_options=/home/misile/arocc/.zig-cache/c/b95a01851fd92449089fe4bbf9631212/options.zig --dep zig --dep build_options -Mbackend=/home/misile/arocc/src/backend.zig -MBuiltins/Builtin.def=/home/misile/arocc/.zig-cache/o/069591fa3ccedaa7b2294e77795fecd4/Builtin.zig -MAttribute/names.def=/home/misile/arocc/.zig-cache/o/082dfe4cf1b80947328d8fd5a9ce5b45/names.zig -MDiagnostics/messages.def=/home/misile/arocc/.zig-cache/o/bb71dae0c7373c62826b4179104e2297/messages.zig -Mzig=/home/misile/arocc/deps/zig/lib.zig -lc --cache-dir /home/misile/arocc/.zig-cache --global-cache-dir /home/misile/.cache/zig --name arocc --zig-lib-dir /usr/lib/zig/9999/lib/ --listen=-
Build Summary: 6/9 steps succeeded; 1 failed
install transitive failure
└─ install arocc transitive failure
   └─ zig build-exe arocc ReleaseFast native 2 errors
error: the following build command failed with exit code 1:
/home/misile/arocc/.zig-cache/o/cee061bb958e6d9b8a5f9d8462ed4a55/build /usr/lib/zig/9999/bin/zig /usr/lib/zig/9999/lib /home/misile/arocc /home/misile/arocc/.zig-cache /home/misile/.cache/zig --seed 0x8f4bafab -Z30eba416d4548a23 -Doptimize=ReleaseFast
```

--- a/src/aro/Attribute.zig
+++ b/src/aro/Attribute.zig
@@ -904,7 +904,7 @@ pub fn applyFunctionAttributes(p: *Parser, ty: Type, attr_buf_start: usize) !Typ
                 else => try p.errStr(.callconv_not_supported, tok, p.tok_ids[tok].lexeme().?),
             },
             .vectorcall => switch (p.comp.target.cpu.arch) {
-                .x86, .aarch64, .aarch64_be, .aarch64_32 => try p.attr_application_buf.append(p.gpa, attr),
+                .x86, .aarch64, .aarch64_be => try p.attr_application_buf.append(p.gpa, attr),
                 else => try p.errStr(.callconv_not_supported, tok, p.tok_ids[tok].lexeme().?),
             },
         },

--- a/src/aro/Compilation.zig
+++ b/src/aro/Compilation.zig
@@ -666,7 +666,7 @@ fn generateBuiltinTypes(comp: *Compilation) !void {
         .arm, .armeb, .thumb, .thumbeb => .{
             .specifier = if (os != .windows and os != .netbsd and os != .openbsd) .uint else .int,
         },
-        .aarch64, .aarch64_be, .aarch64_32 => .{
+        .aarch64, .aarch64_be => .{
             .specifier = if (!os.isDarwin() and os != .netbsd) .uint else .int,
         },
         .x86_64, .x86 => .{ .specifier = if (os == .windows) .ushort else .int },

--- a/src/aro/target.zig
+++ b/src/aro/target.zig
@@ -137,7 +137,7 @@ pub fn float80Type(target: std.Target) ?Type {
 pub fn defaultFunctionAlignment(target: std.Target) u8 {
     return switch (target.cpu.arch) {
         .arm, .armeb => 4,
-        .aarch64, .aarch64_32, .aarch64_be => 4,
+        .aarch64, .aarch64_be => 4,
         .sparc, .sparcel, .sparc64 => 4,
         .riscv64 => 2,
         else => 1,
@@ -327,7 +327,6 @@ pub const FPSemantics = enum {
     pub fn halfPrecisionType(target: std.Target) ?FPSemantics {
         switch (target.cpu.arch) {
             .aarch64,
-            .aarch64_32,
             .aarch64_be,
             .arm,
             .armeb,
@@ -483,7 +482,6 @@ pub fn get32BitArchVariant(target: std.Target) ?std.Target {
         .kalimba,
         .lanai,
         .wasm32,
-        .aarch64_32,
         .spirv,
         .spirv32,
         .loongarch32,
@@ -548,7 +546,6 @@ pub fn get64BitArchVariant(target: std.Target) ?std.Target {
         .x86_64,
         => {}, // Already 64 bit
 
-        .aarch64_32 => copy.cpu.arch = .aarch64,
         .arm => copy.cpu.arch = .aarch64,
         .armeb => copy.cpu.arch = .aarch64_be,
         .loongarch32 => copy.cpu.arch = .loongarch64,
@@ -581,9 +578,8 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
     const llvm_arch = switch (target.cpu.arch) {
         .arm => "arm",
         .armeb => "armeb",
-        .aarch64 => "aarch64",
+        .aarch64 => if (target.abi == .ilp32) "aarch64_32" else "aarch64",
         .aarch64_be => "aarch64_be",
-        .aarch64_32 => "aarch64_32",
         .arc => "arc",
         .avr => "avr",
         .bpfel => "bpfel",
@@ -696,7 +692,7 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
     writer.writeByte('-') catch unreachable;
 
     const llvm_abi = switch (target.abi) {
-        .none => "unknown",
+        .none, .ilp32 => "unknown",
         .gnu => "gnu",
         .gnuabin32 => "gnuabin32",
         .gnuabi64 => "gnuabi64",

